### PR TITLE
SimpleTableView tutorial

### PR DESCRIPTION
**Pull request by kicsipixel**
New: tutorial about cell-based CPTableView

CPTableView with two CPButton and a CPTextfield. User can add and remove new elements.

### CODE REVIEW

**Comment by Dogild** [comment]
I'm wondering if we should put that as an blog article actually ? @mrcarlberg, @primalmotion, @aljungberg though ?

**Comment by mrcarlberg** [comment]
Yeah, should be an excellent blog post!

**Comment by aljungberg** [comment]
Blog posts disappear into the depths after a while, but we do need more tutorials. I'm thinking let's keep it as a tutorial but of course we can announce it in a blog post. :)

File: source/learn/simple-tableview.html.markdown [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+### Code
+
+Let’s define the required instance variables:
+
+	:::objj
+	@implementation AppController : CPObject
+	{
+	    @outlet CPWindow    theWindow;
+	    @outlet CPTableView tableView;
+	    CPMutableArray      nameArray;
```
Comment (Dogild) [inline source/learn/simple-tableview.html.markdown]: You should put _names here

File: source/learn/simple-tableview.html.markdown [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+	}
+
+Our dataSource will be CPMutableArray:
+	
+	:::objj
+	- (id)init
+	{
+	    if (self = [super init]);
+	    {
+	        nameArray = [[CPMutableArray alloc]initWithObjects:"Alpha", "Beta", "Charlie", "Delta"];
```
Comment (Dogild) [inline source/learn/simple-tableview.html.markdown]: You have probably a style issue here, did you use capp_lint ? (space between ] and initWithObjects

File: source/learn/simple-tableview.html.markdown [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+### 'Add’ function
+So far so good, but what if I want to add elements to our list.
+We have to add two objects from the library. One is CPTextField, the second one is CPButton.
+![](simpletableview/simpletableview9.png)
+![](simpletableview/simpletableview11.png)
+
+You can change the CPButton title on the Attributes inspector tab
+![](simpletableview/simpletableview12.png)
+
+Define the CPTextField as an outlet in the Appcontroller. This will help us to get its value.
```
Comment (Dogild) [inline source/learn/simple-tableview.html.markdown]: AppController

**Comment by Dogild** [comment]
Looks good to me. 
@aljungberg can you take a look and merge it ?

File: source/learn/simple-tableview.html.markdown [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+	    @outlet CPTableView tableView;
+	    CPMutableArray      _names;
+	}
+
+Our dataSource will be CPMutableArray:
+	
+	:::objj
+	- (id)init
+	{
+	    if (self = [super init]);
```
Comment (aljungberg) [inline source/learn/simple-tableview.html.markdown]: There shouldn't be a `;` at the end of this line. But don't worry about it, I'll fix it when landing the pull request.

**Comment by aljungberg** [comment]
Published. Thanks, this is great!

**Pull request merged**